README.md

```diff
@@ -37,7 +37,7 @@ http {
 
                 local proxy, err = ws_proxy.new({
                     aggregate_fragments = true,
-                    on_frame = function(origin, typ, payload, last, code)
+                    on_frame = function(origin, role, typ, payload, last, code)
```

Comment:
Ah, you know what, this doc is actually outdated in more ways than one. The `origin` param refers to what is now referenced as `role` in the code (see how the next line says `origin` is "client" or "upstream"). Now, `origin` is not accurate either--the first param to `on_frame` is actually the proxy instance itself:


https://github.com/Kong/lua-resty-websocket-proxy/blob/24678403bd5599a18ba94471aa56780d2437ea3e/lib/resty/websocket/proxy.lua#L337-L338

---

I suggest updating this to rename the first parameter to `proxy` and amending the comment:

```suggestion
                    on_frame = function(proxy, role, typ, payload, last, code)
                        --   proxy: [table]       the proxy instance
```